lib/pavilion/test_config/file_format.py

```diff
@@ -19,6 +19,11 @@ class TestConfigError(ValueError):
 VAR_NAME_RE = re.compile(r'^[a-zA-Z][a-zA-Z0-9_-]*[?+]?$')
 
 
+class PathElem(yc.CategoryElem):
+    """This is for category elements that need a valid unix path regex."""
+    _NAME_RE = re.compile(r"([^ !$`&*()+]|(\\[ !$`&*()+]))+")
```

Comment:
As per my comments in chat, this can just be r".+$". There are some corner cases, but nothing worth checking for..